core/sql/comexe/ComTdbExeUtil.cpp

```diff
@@ -2417,6 +2425,15 @@ short ComTdbExeUtilLobShowddl::getLOBnum(short i)
   return lobNum;
 }
 
+NABoolean ComTdbExeUtilLobShowddl::getIsExternalLobCol(short i)
+{
+  if ((i > numLOBs_) || (i <= 0))
+    return -1;
```

Comment:
Does -1 fit in an NABoolean? Would it be evaluated as "TRUE"? I'm wondering if NABoolean is the right  return type for this method.